### DISCUSSION TITLE
get_gene_activity

### DISCUSSION THREAD

**Discussion by TingTingShao** [aertslab/pycisTopic#146] [Q&A]
Hi, 

Could I please ask how get_gene_activity works. I can see it is calculated on the consensus regions. So the gene activity score is mostly distance based if I understand it correctly. The score of same gene for different cell/nuclei, I suppose it is different, and the difference is based on the openness of the region in each cell/nuclei?

Thanks,
tingting

**Comment by SeppeDeWinter** [comment]
Hi TingTing

This is based on this paper: https://www.nature.com/articles/s41586-021-04262-z#Sec8
See methods: "Gene accessibility matrix".

In summary, gene activity is based on:

- distance to the gene.
- Specificity of the peaks (gini score across cells).
- and accessibility of the peaks (per cell).

Does this help?

All the best,

Seppe

**Comment by TingTingShao** [comment]
Yes, it clarifies the question.

Thanks!
tingting
